test/e2e/mongodbtests/mongodbtests.go

```diff
@@ -363,14 +363,17 @@ func BasicFunctionality(mdb *mdbv1.MongoDBCommunity) func(*testing.T) {
 		t.Run("Service Set Has OwnerReference", ServiceHasOwnerReference(mdb, mdbOwnerReference))
 		t.Run("Agent Secrets Have OwnerReference", AgentSecretsHaveOwnerReference(mdb, mdbOwnerReference))
 		t.Run("Connection string secrets are configured", ConnectionStringSecretsAreConfigured(mdb, mdbOwnerReference))
-		t.Run("Test Status Was Updated", Status(mdb,
-			mdbv1.MongoDBCommunityStatus{
-				MongoURI:                   mdb.MongoURI(""),
-				Phase:                      mdbv1.Running,
-				Version:                    mdb.GetMongoDBVersion(),
-				CurrentMongoDBMembers:      mdb.Spec.Members,
-				CurrentStatefulSetReplicas: mdb.Spec.Members,
-			}))
+		// TODO: this is temporary, remove the need for skipStatuscheck after 0.7.4 operator release
+		if len(skipStatusCheck) > 0 && !skipStatusCheck[0] {
```

Comment:
This check is not present in the latest operator release, hence it would fail on `0.7.3` operator install. This is temporary and would be removed after the next operator release.